Editor/Volume/LightingVolumeEditor.cs

```diff
@@ -24,9 +24,9 @@ public override void OnEnable()
 
         public override void OnInspectorGUI()
         {
-            PropertyField(m_LightingIsEnabled, EditorGUIUtility.TrTextContent("Lighting Is Enabled"));
-            PropertyField(m_BakedLightingMultipler, EditorGUIUtility.TrTextContent("Baked Lighting Multiplier"));
-            PropertyField(m_VertexColorLightingMultiplier, EditorGUIUtility.TrTextContent("Vertex Color Lighting Multiplier"));
+            PropertyField(m_LightingIsEnabled, EditorGUIUtility.TrTextContent("Lighting Is Enabled", "Controls whether lighting is enabled or disabled."));
+            PropertyField(m_BakedLightingMultipler, EditorGUIUtility.TrTextContent("Baked Lighting Multiplier", "Controls the strength of the baked lighting information when applied to the mesh."));
```

Comment:
Super minor note here: I would recommend using the word "intensity" or "magnitude" or "brightness" rather than "strength". "strength" can sometimes be associated with contrast.

I would also recommend using the word "surface" instead of "mesh" since mesh makes it sound like we are performing transformations to the mesh data, when all we are doing is modifying surface data when rendering.